src/main/java/LoginDelocalized/AuthenticationService.java

```
@@ -3,19 +3,29 @@
 public class AuthenticationService {
 
     public User createUser(String username, String password) {
-
+        return new User(username, password);
     }
 
     public User changeUsername(User user, String newUsername) {
-
+        if (newUsername == null || newUsername.isEmpty() || newUsername.trim().isEmpty()) {
+            System.out.println("Please provide valid Username");
+        } else {
+            user.setName(newUsername);
+        }
```

Depending on how this is used, it might be better to throw an exception.